simulator/mips/t/unit_test.cpp

```diff
@@ -243,3 +206,454 @@ TEST_CASE( "MIPS32_instr_disasm: Process_Disasm_Branches")
     CHECK(MIPS32Instr(0x0c0004d2).get_disasm() == "jal 0x4d2");
 }
 
+
+// Converted SPIM TT tests
+// ************************* R INSTRUCTIONS ****************************
+// 19 done
+
+// Not implemented in SPIM TT
+TEST_CASE ( "MIPS32_instr: sll")
+{
+    MIPS32Instr instr( "sll");
```

Comment:
Please move disassembly checks for implemented instructions inside one of the test cases.
With that, we can easily track what is not implemented yet.

Reply:
Implemented in unit_test.cpp?
Should I write (copy) checks like this
    CHECK(MIPS32Instr(0x01398820).get_disasm() == "add $s1, $t1, $t9");
in alphabet order?
One more thing: now build requires module builtins from pyhon-future.

Reply:
> now build requires module builtins from pyhon-future.

Yes, I'm aware. Please run `pip install future`

> Should I write (copy) checks like this in alphabet order?

Please move those checks only if you have implemented the "individual" tests. You would have something like:

```c++
TEST_CASE ("add") {
    CHECK(MIPS32Instr(0x01398820).get_disasm() == "add $s1, $t1, $t9");
    MIPS32Instr instr("add");
// your tests here
}
```

and the long list of instruction without "individual tests" — so we can see which tests have to be added.